src/objects/aff/zcl_abapgit_object_uipg.clas.abap

```diff
@@ -0,0 +1,32 @@
+CLASS zcl_abapgit_object_uipg DEFINITION
+  PUBLIC
+  INHERITING FROM zcl_abapgit_object_common_aff
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    METHODS zif_abapgit_object~changed_by
+        REDEFINITION .
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_OBJECT_UIPG IMPLEMENTATION.
+
+
+  METHOD zif_abapgit_object~changed_by.
+    TRY.
+        DATA(lo_api) = NEW /ui2/cl_uist_sval_sql( ).
+        DATA(ls_metadata) = /ui2/cl_uipg_db_access=>get_instance( )->read_wb_metadata( iv_page_id  = CONV #( ms_item-obj_name )
```

Comment:
- No CONV (740)
- class must be called dynamically since it doesn't exist in 702